contracts/CirculatingSupply.sol

```diff
@@ -17,4 +18,10 @@ contract CirculatingSupply is Ownable {
     function get() public view returns (uint256) {
         return circulatingSupply;
     }
+
+    function setUp(uint256 _circulatingSupply) public {
```

Comment:
Setup function should take `bytes memory` as a parameter.
```solidity
    function setUp(bytes memory initializeParams) public override {
        require(!initialized, "Guard is already initialized");
        initialized = true;
        (uint256 _circulatingSupply) = abi.decode(initializeParams, (uint256));
   ```
   
   This is to ensure it is compatible with our module factory.